src/mimefactory.rs

```diff
@@ -349,7 +349,7 @@ impl<'a, 'b> MimeFactory<'a, 'b> {
                         .stock_str(StockMessage::AcSetupMsgSubject)
                         .await
                         .into_owned()
-                } else if chat.typ == Chattype::Group {
+                } else if chat.typ == Chattype::Group || chat.typ == Chattype::Mailinglist {
```

Comment:
We don't send messages to the mailing list, so it does not make sense to add `Re` or not here, this should never be called anyway.

Probably transform this whole if into `match`, with explicit branches for `Group`, `Single` (currently the else branch) and a `Mailinglist` with some fallback, probably just `chat.name`.

Reply:
i just remove the change, mimefactory is not called for mailinglists at all currently, right. we can refactor that as needed, there is also some pending work wrt subjects, so there seems not to be much need to do this in scope of this pr (also to make the mailinglists2 pr not even bigger :)